src/funcCoreTools/funcHostTask.ts

```diff
@@ -79,34 +68,57 @@ export function registerFuncHostTaskEvents(): void {
         // Used to stop the task started with pickFuncProcess.ts startFuncProcessFromApi.
         if (DotnetDebugDebugConfiguration.is(debugSession.configuration) && debugSession.configuration.launchServiceData.buildPath) {
             const buildPathUri: vscode.Uri = vscode.Uri.file(debugSession.configuration.launchServiceData.buildPath)
-            stopFuncTaskIfRunning(buildPathUri.fsPath, /* terminate */ true)
+
+            const workspaceFolder = buildPathToWorkspaceFolderMap.get(debugSession.configuration.launchServiceData.buildPath)
+            if (workspaceFolder === undefined) {
+                throw Error()
```

Comment:
nit: I wasn't sure if throwing here is correct.

Maybe a `vscode.window.showWarningMessage('Unable to find workspaceFolder for debugConfiguration.buildPath");` + however its localized